app/models/job.rb

```diff
@@ -1,5 +1,28 @@
 # frozen_string_literal: true
 
 class Job < ApplicationRecord
-  validates :company, :title, :description, :link, :location, :image_url, presence: true
+  SPONSORSHIP_LEVELS = { RUBY: 3, EMERALD: 2, SAPPHIRE: 1, OPAL: 0 }.freeze
```

Comment:
This is more for readability than for functionality - I want to have a clear mapping of our existing sponsorship levels to the integer levels in the database so that I can easily remember them and make sure I'm putting the right level on new job listings. Let men know if there's a better way to do this!

Reply:
you could go with defining it as an [enum](https://api.rubyonrails.org/v5.2.4.4/classes/ActiveRecord/Enum.html) if you want to lean into using the names of the levels around the codebase instead of the int values 🙂

Reply:
:-OOOO My mind is blown.